.github/workflows/download-pulumi-cron.yml

```diff
@@ -103,6 +103,36 @@ jobs:
         run: |
           echo "Expected version ${{ steps.vars.outputs.expected-version }} but found ${{ steps.vars.outputs.installed-version }}"
           exit 1
+  windows-winget-install:
+    name: Install Pulumi with WinGet on Windows
+    runs-on: windows-latest
+    steps:
+      - name: Remove Pre-installed Pulumi
+        run: |
+          if command -v "pulumi.exe"; then
+            PULUMI_INSTALL_DIR=$(dirname "$(command -v "pulumi.exe")")
+            echo "Deleting Pulumi"
+            rm -v "$PULUMI_INSTALL_DIR"/pulumi*
+          fi
+      - name: Install winget
+        uses: Cyberboss/install-winget@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install Pulumi Using WinGet
+        run: winget install pulumi --disable-interactivity --accept-source-agreements --verbose
+      - run: echo "C:/Program Files (x86)/Pulumi" >> "${GITHUB_PATH}"
```

Comment:
It's strange that we're installing x64 binaries in `Program Files (x86)`. I opened https://github.com/pulumi/pulumi-winget/issues/30 to consider installing them where they _should_ go, e.g. `Program Files`.